pom.xml

```diff
@@ -54,7 +54,9 @@
 
         <maven.compiler.parameters>true</maven.compiler.parameters>
 
+        <!-- Include quarkus.platform.test.version to allow different dependency build artifacts -->
         <quarkus.platform.version>2.7.5.Final</quarkus.platform.version>
+        <quarkus.platform.test.version>2.7.5.Final</quarkus.platform.test.version>
```

Comment:
Minor suggestion would be to use `${quarkus.platform.version}` as the default for the test version.

Reply:
Feel free to ignore :-)